drivers/hid/hidclass/fdo.c

```diff
@@ -1,28 +1,32 @@
 /*
  * PROJECT:     ReactOS Universal Serial Bus Human Interface Device Driver
- * LICENSE:     GPL - See COPYING in the top level directory
- * FILE:        drivers/hid/hidclass/fdo.c
+ * LICENSE:     GPL-3.0-or-later (https://spdx.org/licenses/GPL-3.0-or-later)
  * PURPOSE:     HID Class Driver
- * PROGRAMMERS:
- *              Michael Martin (michael.martin@reactos.org)
- *              Johannes Anderwald (johannes.anderwald@reactos.org)
+ * COPYRIGHT:   Copyright  Michael Martin <michael.martin@reactos.org>
+ *              Copyright  Johannes Anderwald <johannes.anderwald@reactos.org>
+ *              Copyright 2022 Roman Masanin <36927roma@gmail.com>
  */
 
 #include "precomp.h"
 
 #define NDEBUG
 #include <debug.h>
 
+IO_COMPLETION_ROUTINE HidClassFDO_QueryCapabilitiesCompletionRoutine;
+IO_COMPLETION_ROUTINE HidClassFDO_DispatchRequestSynchronousCompletion;
+IO_COMPLETION_ROUTINE HidClassFDO_ReadCompletion;
```

Comment:
Please, remove this change. Not required

Reply:
they do, because these functions is a driverIO callbacks, MSDN and VS suggest to declare these functions like that before definition.

Reply:
Please don't add that to ReactOS code.

Reply:
We absolutely want this in ReactOS code. We always want to fix our drivers so they correctly compile with WDK.